aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/APIGatewayV2ProxyResponseEvent.java

```diff
@@ -0,0 +1,99 @@
+package com.amazonaws.services.lambda.runtime.events;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * @author Tim Gustafson <tjg@amazon.com>
+ */
+public class APIGatewayV2ProxyResponseEvent implements Serializable, Cloneable {
```

Comment:
Suggest changing names to be like `APIGatewayWebSocketProxy....`

Reply:
I was following the naming convention of the product.  All the SDKs refer to the product as "APIGatewayV2".  I agree, it's confusing, but it maps to the actual product implementation.

For example:

https://docs.aws.amazon.com/cli/latest/reference/apigatewayv2/index.html

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-api.html

https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/apigatewayv2/package-summary.html